front/src/reducers/osrdconf/stdcmConf/stdcmConfReducers.spec.ts

```diff
@@ -113,3 +115,116 @@ describe('stdcmConfReducers', () => {
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(state.pathSteps[0]).toEqual({
+      ...newOrigin,
+      arrival: undefined,
+      arrivalType: ArrivalTimeTypes.ASAP,
+      arrivalToleranceBefore: undefined,
+      arrivalToleranceAfter: undefined,
+    });
+  });
+
+  it('should handle updateOriginArrivalType', () => {
+    store.dispatch(stdcmConfSliceActions.updateOriginArrivalType(ArrivalTimeTypes.PRECISE_TIME));
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(state.pathSteps[0]).toEqual({
+      ...newOrigin,
+    });
+  });
+
+  it('should handle updateOriginTolerances', () => {
+    store.dispatch(
+      stdcmConfSliceActions.updateOriginTolerances({ toleranceBefore: 300, toleranceAfter: 300 })
+    );
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(state.pathSteps[0]).toEqual({
+      ...newOrigin,
+      arrivalToleranceBefore: 300,
+      arrivalToleranceAfter: 300,
+    });
+  });
+});
+
+describe('Destination arrival updates', () => {
+  const newDestination = {
+    ...last(testDataBuilder.buildPathSteps())!,
+    arrivalType: ArrivalTimeTypes.ASAP,
+  };
+  const store = createStore(initialStateSTDCMConfig);
+
+  beforeEach(() => {
+    store.dispatch(stdcmConfSliceActions.updateDestinationV2(newDestination));
+  });
+
+  it('should handle updateDestinationArrival', () => {
+    store.dispatch(stdcmConfSliceActions.updateDestinationArrival('2024-08-12T15:45:00.000+02:00'));
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(last(state.pathSteps)).toEqual({
+      ...newDestination,
+      arrival: '2024-08-12T15:45:00.000+02:00',
+      arrivalType: ArrivalTimeTypes.PRECISE_TIME,
+    });
+  });
+
+  it('should handle updateDestinationArrival with undefined', () => {
+    store.dispatch(stdcmConfSliceActions.updateDestinationArrival(undefined));
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(last(state.pathSteps)).toEqual({
+      ...newDestination,
+      arrival: undefined,
+      arrivalType: ArrivalTimeTypes.ASAP,
+      arrivalToleranceBefore: undefined,
+      arrivalToleranceAfter: undefined,
+    });
+  });
+
+  it('should handle updateDestinationArrivalType', () => {
+    store.dispatch(
+      stdcmConfSliceActions.updateDestinationArrivalType(ArrivalTimeTypes.PRECISE_TIME)
+    );
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(last(state.pathSteps)).toEqual({
+      ...newDestination,
+      arrivalType: ArrivalTimeTypes.PRECISE_TIME,
+    });
+  });
+
+  it('should handle updateDestinationTolerances', () => {
+    store.dispatch(
+      stdcmConfSliceActions.updateDestinationTolerances({
+        toleranceBefore: 300,
+        toleranceAfter: 300,
+      })
+    );
+    const state = store.getState()[stdcmConfSlice.name];
+    expect(last(state.pathSteps)).toEqual({
+      ...newDestination,
+      arrivalToleranceBefore: 300,
+      arrivalToleranceAfter: 300,
+    });
+  });
+});
```

Comment:
Could you move this code inside `describe('stdcmConfReducers'`? Just before `testCommonConfReducers(stdcmConfSlice);`. This way, when the tests are run, it will be clear that these new cases are related to the stdcmConfReducers.